app/mailers/mailbot.rb

```diff
@@ -80,6 +80,20 @@ def send_on_call_for_papers_dates_updates(conference)
     end
   end
 
+  def notification_email(conference, event, comment)
+    recipients = User.joins(:roles).where('roles.name IN (?)', [:organizer, :cfp]).where('roles.resource_id = ?', conference.id).select(:name, :email).distinct
```

Comment:
For educative purposes (as in, you haven't done anything wrong, I am just suggesting easier/faster/different options):
1) for the roles we use Rolify gem, which includes the following capability:
User.with_role :organizer, Conference.second

More details: https://github.com/RolifyCommunity/rolify/wiki/Usage

2) You can use where like this too:
Conference.where(title: 'test', short_title: 'test')
or
Conference.where(id: [3,4])

More details: http://guides.rubyonrails.org/active_record_querying.html